klaas/scripts/train_disp_regressor.py

```diff
@@ -76,18 +84,21 @@ def main(configuration_path, signal_path, predictions_path, disp_model_path, sig
         key=key,
         columns=columns_to_read,
     )
-
     log.info('Total number of events: {}'.format(len(df)))
+
+    source_x, source_y = horizontal_to_camera(
+        az=df[az_source_col], zd=df[zd_source_col],
+        az_pointing=df[az_pointing_col], zd_pointing=df[zd_pointing_col],
+    )
+
     df['true_disp'] = euclidean_distance(
```

Comment:
I think I remember having different naming conventions for true and predicted values. Was it ´disp_true´ and ´disp_prediction´  or something?

Reply:
Those are not saved, just internally used in this script

Reply:
The values that are store are 

* `source_x_prediction`
* `source_y_prediction`
* `disp_prediction`

Which imho comply to our naming conventions